test/remote/gateways/remote_reach_test.rb

```diff
@@ -74,6 +75,88 @@ def test_failed_capture
     assert_equal 'Not Found', response.message
   end
 
+  def test_successful_refund
+    response = @gateway.refund(
+      @amount,
+      '5cd04b6a-7189-4a71-a335-faea4de9e11d',
+      { reference_id: 'REFUND_TAG' }
+    )
+
+    assert_success response
+    assert response.params['response'][:RefundId].present?
```

Comment:
Hmm, not sure why we are getting mixed symbols and strings in the response. Based on the `parse` method, all of the contents of the response hash _should_ be symbolized, but it seems like that is being undone incidentally elsewhere?

Reply:
good catch, I change them to symbols